src/routes/ProductsRoute.ts

```diff
@@ -0,0 +1,7 @@
+import { Router } from 'express';
+import { getProducts } from '../controllers/ProductController';
+
+const router: Router = Router();
```

Comment:
router type can be inferred from the return value of `Roiuter` no need to declare it and please leave an empty line